Sources/PackageModel/Manifest.swift

```diff
@@ -42,6 +42,9 @@ public final class Manifest: ObjectIdentifierProtocol, CustomStringConvertible,
     /// The name of the package.
     public let name: String
 
+    /// Whether kind of package this manifest is from.
+    public let kind: PackageReference.Kind
```

Comment:
I would call this packageKind otherwise it implies this is "manifest kind" which isn't a thing.

Reply:
Totally agree. Fixed.